chrome/skin/style-linux.css

```diff
@@ -0,0 +1,6 @@
+@import 'style.css';
```

Comment:
@kaganjd I'm not sure we want to use '@import' because we'll need to manually unload the stylesheets when the addon is disabled/uninstalled.

Would you mind manually loading / unloading both the main style.css file and the platform-specific files?